libs/libopen62541/Makefile

```
@@ -0,0 +1,103 @@
+#
+# Copyright (C) 2023 Michele Primavera
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libopen62541
+PKG_VERSION:=1.3.6
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/open62541/open62541.git
+PKG_SOURCE_VERSION:=v$(PKG_VERSION)
+PKG_MIRROR_HASH:=1587eba9e697591a342d91df02acb7460e74617895f810c006ad3d7e4bce21b8
+
+PKG_LICENSE:=MPL-2.0
+PKG_LICENSE_FILES:=LICENSE
```

```suggestion
PKG_LICENSE_FILES:=LICENSE
PKG_MAINTAINER:=Michele Primavera <primavera@elmod.it>
```

it would be nice to add yourself as the maintainer here

---

I will provide updates for this package but I cannot guarantee they will be in a timely manner, that's why I didn't specify myself there

---

oh thank you for explaining. added!